agot-bg-game-server/src/common/ingame-game-state/westeros-game-state/wildlings-attack-game-state/WildlingsAttackGameState.ts

```diff
@@ -107,9 +107,21 @@ export default class WildlingsAttackGameState extends GameState<WesterosGameStat
     }
 
     firstStart(wildlingStrength: number, participatingHouses: House[] = []): void {
+        if (this.game.wildlingStrength != wildlingStrength) {
+            this.game.wildlingStrength = wildlingStrength;
+            this.entireGame.broadcastToClients({
+                type: "change-wildling-strength",
+                wildlingStrength: wildlingStrength
+            });
+        }
+
```

Comment:
Why not sending this inside `executeNightsWatchWon` since it's where it's changed?

Reply:
I guess I wanted to generalize this as the given value for the wildlings attack should then always be represented in `game`. But of course I can change that.